daffodil-test/src/test/resources/org/apache/daffodil/section07/variables/variables.tdml

```diff
@@ -24,7 +24,7 @@
   xmlns:fn="http://www.w3.org/2005/xpath-functions"
   xmlns:ex="http://example.com"
   defaultRoundTrip="true">
-  
+
```

Comment:
In general, I find it's best to avoid these kinds of unrelated whitespace changes to make reviewing the actual changes easier. Sometimes this requires tweaks to IDE settings to disable this. We probably should fix the whitespace in our TDML files at some point, but we should that to everything at once like we did when we standardized our scala formatting.